android/source/VideoLocker/AndroidManifest.xml

```diff
@@ -15,7 +15,6 @@
     <uses-permission android:name="android.permission.ACCESS_WIFI_STATE" />
     <uses-permission android:name="android.permission.DOWNLOAD_WITHOUT_NOTIFICATION" />
     <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
-    <uses-permission android:name="android.permission.WRITE_SETTINGS"/>
```

Comment:
I am not sure if this permission is required by the app. I will take this change through testing to ensure all third party calls work without this permission.